python-package/xgboost/core.py

```diff
@@ -409,6 +410,14 @@ def _maybe_dlpack_data(data, feature_names, feature_types):
     data = fromDlpack(data)
     return data, feature_names, feature_types
 
+def _maybe_chunked_array(data, feature_names, feature_types):
+    if PYARROW_INSTALLED and isinstance(data, ArrowChunkedArray):
+        if not PANDAS_INSTALLED:
+            raise Exception(('pandas must be available to use this method.'
```

Comment:
I suppose better to say: 'Pandas must be available to use Apache Arrow as an input format'

Reply:
Thanks. Will incorporate your input.